README.md

```diff
@@ -1,6 +1,160 @@
-<<<<<<< HEAD
-# webpack
-An Exercise to setup a Frontend development Environment using Webpack
-=======
-# webpack
->>>>>>> 1e3f6b863829ffcc4f663cdb23af81494ccbd57d
+<a name="readme-top"></a>
+
+  <h3><b>Webpack Boiler plate for frontend development/Todo List</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
```

Comment:
- [ ] It would be nice if you fix your FAQ link because right now it is broken 😨 Kindly fix it to make your readme descriptive and professional